cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -277,4 +276,44 @@ export class MigrationWave {
         cy.contains("li", date.toLocaleString("en-us", { month: "long" })).click();
         cy.contains("button:not([disabled])", `${date.getDate()}`).first().click();
     }
+
+    public clickOnApplicationStatus() {
+        cy.contains(this.name)
+            .parents("tr")
+            .within(() => {
+                cy.get(MigrationWaveView.applicationStatusColumn).click();
```

Comment:
The selector should also be renamed to something more appropriate like 'waveStatusColumn' .